src/gConsent/discover/redirectToAccessManagementUi.ts

```diff
@@ -20,9 +20,9 @@
 //
 
 import type { UrlString, WebId } from "@inrupt/solid-client";
-import type {
-  DatasetWithId,
-  VerifiableCredential,
+import {
+  type DatasetWithId,
+  type VerifiableCredential,
```

Comment:
Is this change intentional, or is it introduced by `prettier`?

Reply:
It wasn't intentional at all. I believe it has been introduced by `prettier` indeed